docs/TESTS.md

```diff
@@ -1,2 +1,16 @@
 # Tests
 
+We use the `lib.debug.runTests` function from the Nixpkgs standard library to run the tests.
+Nixpkgs should be automatically set up when you install Nix.
+
+## Running the Tests
+To run the tests, navigate to the exercise folder using `cd {exercise-folder-location}`.
+Then run the following command:
+```bash
```

Comment:
I recently learned that you can also use `shell` as the code block's type

```suggestion
```shell
```